COMMUNICATION.md

```diff
@@ -0,0 +1,63 @@
+# Communicating within the iOS team
+
+## Why we have guidelines about communicating
+Within our iOS team we mostly communicate with each other via the magic of the internet, and that is great! It's easy and very efficient. But online communication can easily result into misunderstandings, which tend to be long, time consuming and inefficient. That's why we created some guidelines for how we like to communicate professionally, keep them in the back of your head when commenting on GitHub or Slack.
+
+## When to pay attention to the guidelines
+Obviously we're communicating all the time. We can't read this document every time we're having something come out of our mouths or keyboards. That would result in silencing the world 🤐, and thats the opposite of the intention of these guidelines.
+
+- **Only in professional communication**
+When making jokes, talking about your favourite tea, or planning a team members birthday party surely a lot of things can go wrong in communication as well. But these guidelines are meant to solve issues in our professional communication, like pull request comments, code review and more thorough discussions.
+
+- **When writing short replies**
+When you write a reply of less than ~5 words, have an alarm bell in the back of your head 🔔. Is it just tacit agreement? Then it’s probably fine! “Great job”, “Couldn’t have put it better”,  “toats awesome!” are all fine. In every other case though: 🚨!
+
+It good practice to elaborate. It may seem inefficient to be so verbose in our replies, but it will avoid misunderstandings, which can result in way greater inefficiency.
+
+## Criticising or asking questions
+Start by repeating **what** you are replying to in your own words. This forces you to indulge yourself into the perspective of the other, which creates a greater understanding of the issue both for yourself and the other when reading your reply.
+
+### To be avoided:
+- Don't use optional here. 
+- What’s your suggestion?
+
+### Preferred:
+- **I see you are using an optional as an array of User object in the example above**, wouldn’t it be better to have that property default to an empty array?
```

Comment:
Instead of:
>wouldn’t it be better to have that property default to an empty array?

I would go for:
>have you consider to have that property default to an empty array?

There's a chance that it was already considered and you might end up learning something on why it wasn't done that way. Meanwhile on the first one you are suggesting they didn't know something or tried something in advance without asking.